README.md

```diff
@@ -17,16 +17,112 @@ pip install -r requirements.txt
 pre-commit install
 ```
 ## linux
-make sure to have pip
+use the `make` command with an `action` when developing on linux.  think of `actions` as a list of predefined commands to help simplify common development routines.
+
+#### view avaiable actions
+`make help` to see all avaiable actions
+
 ```sh
-sudo apt install python3-pip
+public-api$ make help

```

Comment:
error occured
```
Command 'make' not found, but can be installed with:
sudo apt install make        # version 4.3-4.1build1, or
sudo apt install make-guile  # version 4.3-4.1build1
```

Reply:
ohh the make command may not be included in all gnu gcc pacakges.

Reply:
i added an entire section for the make package install instructions. should be resolved now